src/main/java/de/tum/in/www1/artemis/web/rest/dto/LectureUnitSplitDTO.java

```diff
@@ -0,0 +1,49 @@
+package de.tum.in.www1.artemis.web.rest.dto;
+
+import java.time.ZonedDateTime;
+
+public class LectureUnitSplitDTO {
```

Comment:
JavaDoc explanations of this file would be helpful. I think this DTO could also be rewritten using Java record ?

Reply:
Changed to Java record and added JavaDoc explanation